README.md

```diff
@@ -37,21 +36,97 @@ Deployment of NVA consists of 2 templates which is used to create resources in 2
   | OrcidClientSecret | Secret to authenticate for ORCID | {{secretsmanager:OrcidClientSecret}} |
   | AlmaSruHost | URL to ALMA Library services | {{secretsmanager:AlmaSruHost}} |
   
-  ---
-**Current modules in NVA API and their deployment**
-
-|API path|GIT repository|SAR App Name|Version|Notes|
-|-----|-----|-----|-----|-----|
-|/alma|nva-alma-proxy|SruLastPublication|0.1.2| |
-|/upload|nva-upload-multipart|UploadMultipart|0.1.6| |
-|/person|nva-bare-proxy|PersonData|0.1.3| |
-|/publication|nva-publication-api|NvaPublicationApi|0.1.11| |
-|/channel|nva-channel-registry|PublicationChannelRegister|0.1.2| |
-|/doi-fetch|nva-fetch-doi|nva-fetch-doi|0.1.7| |
-|/doi-requests|nva-doi-requests-api|NvaDoiRequestsApi|0.1.0| |
-|/project|nva-cristin-projects|Projects|0.1.3| |
-|/institution|nva-institution-proxy|NvaInstitutionProxy|0.1.3| |
-|/download|nva-download-file|NvaDownloadPublicationFileApi|0.1.4| |
-|/customer|nva-customer-api|NvaCustomerApi|0.1.3|2020-09-09| |
-|/users-roles|nva-user-access-service|NvaUsersAndRolesService|0.1.2|2020-09-09| |
-| |nva-cognito-post-authentication-trigger|NvaCognitoPostAuthenticationTrigger|0.1.1|2020-09-09| |
+***
+
+## Deployment steps
+
+ 1. Identify repositories with changes.
```

Comment:
Perhaps we should clarify that step 1-4 can be done any time before deployment, not as a part of the deployment process. Making artifacts ready to deploy doesn't imply immidiate deployment.

Reply:
I believe you don't do step 3 and onwards before you ship to test or production.

Reply:
So I agree that probably good to define what is assembling of artifacts (CI) and what is part of processing it for delivery (CD)